client/components/_tests_/footer.test.tsx

```diff
@@ -0,0 +1,11 @@
+import renderer from "react-test-renderer";
+import Footer from "../footer";
+
+describe("footer test", () =>{
+    it('is redendring properly footer component ', () => {
+        const tree = renderer.create(
+            <Footer />
+        ).toJSON()
+        expect(tree).toMatchSnapshot();
```

Comment:
I guess you could just check if the `id` links exist inside of footer.

you don't need to check the whole structure, because this test will be so fragile  :/

Reply:
![Screenshot from 2023-04-24 18-57-05](https://user-images.githubusercontent.com/106893704/234125471-d98b5b3f-4876-4956-8a53-ddf7fa754536.png)

you mean, i create a `ID` for footer and test if this `ID` is being returned?

Reply:
Yep, I think, it's fine

Reply:
Ok then, i'm getting now that isn't so good use snapshot for these components(or every component), every time i update i forgot to to update the snap as well and then i have to reset my commit and force a new one :´)

Reply:
dude what a hit on my head XD, is being harder than i thought.